sites/public/src/RenderIf.tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouter } from "next/router"
+
+const RenderIf = (props: { language: string; children: JSX.Element }) => {
+  const router = useRouter()
+
+  if (props.language == "all") {
```

Comment:
Looks like these three ifs can be combined into one if they all return the same thing